pkg/webui/components/panel/toggle/index.js

```diff
@@ -0,0 +1,56 @@
+// Copyright © 2023 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import classnames from 'classnames'
+
+import Button from '@ttn-lw/components/button'
+
+import PropTypes from '@ttn-lw/lib/prop-types'
+
+import styles from './toggle.styl'
+
+const Toggle = ({ options, onToggleChange, active }) => (
+  <div className={classnames(styles.toggle, 'd-flex j-center al-center gap-cs-xxs p-cs-xxs')}>
```

Comment:
```suggestion
  <div className={classnames(styles.toggle)}>
```
Let's not mix utility and normal class definitions too much. This makes reasoning about the styles harder. Here, you could just define everything in the class.